src/config/interface.ts

```diff
@@ -30,6 +30,12 @@ export interface auth_status {
   is_verified: boolean;
```

Comment:
Can you refactor the `auth_status` interface to not have the `idp` field anymore? It'll always end up as Auth0 anyway so storing that information is irrelevant. You may need to update the backend at a later point as well if its passing the extra field to the front-end.